Source/Core/Core/Config/AchievementSettings.cpp

```diff
@@ -0,0 +1,17 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "Core/Config/AchievementSettings.h"
+
+#include <string>
+
+#include "Common/Config/Config.h"
+
+namespace Config
+{
+// Configuration Information
+const Info<bool> RA_INTEGRATION_ENABLED{{System::Achievements, "General", "IntegrationEnabled"},
```

Comment:
`INTEGRATION` here seems redundant. Why not just `ENABLED`?

Reply:
It's possible, though there are several components to this that get enabled or disabled separately so I wanted to differentiate this from that.